tool/grind.dart

```diff
@@ -959,24 +959,25 @@ Future<void> testDartdocFlutterPlugin() async {
 @Task('Validate the SDK doc build.')
 @Depends(buildSdkDocs)
 void validateSdkDocs() {
-  const expectedLibCount = 0;
-  const expectedSubLibCount = 19;
+  const expectedLibCounts = [0, 1];
```

Comment:
Perhaps add a comment explaining why we're tolerant of both 19 and 20 core libs?

Reply:
done